package/python-typing/python-typing.mk

```diff
@@ -4,11 +4,11 @@
 #
 ################################################################################
 
-PYTHON_TYPING_VERSION = 3.6.6
+PYTHON_TYPING_VERSION = 3.7.4.1
```

Comment:
doesn't the pypi typing module not do anything on python >3.5? Per its readme page [on pypi](https://pypi.org/project/typing/)